src/test/java/org/opensearch/security/SlowIntegrationTests.java

```diff
@@ -216,34 +210,4 @@ public void testNodeClientDisallowedWithNonServerCertificate2() throws Exception
             Assert.fail(e.toString());
         }
     }
-
-    @Test
-    public void testDelayInSecurityIndexInitialization() throws Exception {
```

Comment:
This test depends on cluster timeout being produced due to red cluster state which is highly dependent on how fast machine is. Timeout is introduced here: https://github.com/opensearch-project/security/blob/9be1b272f99421cdf7470cf30c44e11f1061f371/src/test/java/org/opensearch/security/test/helper/cluster/ClusterHelper.java#L122C1-L125C6
Decided to get rid of it since it's highly unstable

Reply:
Removing is a good approach to fix the near term issues.

Could you create an issue to track recreating this test scenario?  IMO its an interesting case for making sure that clusters can recover when the security plugin in in the mix.

Reply:
Here's new issue since this is outside of scope of this task: 
https://github.com/opensearch-project/security/issues/3772